types/yandex-metrika-tag/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json", 
+    "rules": {
+        "dt-header": false,
```

Comment:
Please follow instructions in our contribution guide and remove these lines that disable the relevant tslint rules.

Reply:
Done.